libs/langchain/langchain/chains/mapreduce.py

```diff
@@ -23,6 +24,16 @@
 from langchain.chains.llm import LLMChain
 
 
+@deprecated(
+    since="0.2.13",
+    removal="0.4.0",
```

Comment:
seeing 1.0 for some stuff and 0.4 for other

Reply:
thanks